src/index.js

```diff
@@ -116,6 +121,21 @@ class CreditCardInput extends Component {
     }
   }
 
+  validateExpirationDate (date) {
+    const expiry = date.split(' / ')
+    if (expiry.length < 2) {
+        return
+    }
+    let year = Number(expiry[1])
+    let month = Number(expiry[0])
+    let newDate = "20" + year + " " + month
+
+    if (moment(newDate).isValid()) {
```

Comment:
Perhaps you could just do `isValid(new Date(year, month))` - think that could work!